torch/distributions/kl.py

```diff
@@ -258,13 +258,21 @@ def _kl_pareto_pareto(p, q):
     return result
 
 
+@register_kl(Poisson, Poisson)
+def _kl_poisson_poisson(p, q):
+    return p.rate * (p.rate.log() - q.rate.log()) - (p.rate - q.rate)
+
+
 @register_kl(Uniform, Uniform)
 def _kl_uniform_uniform(p, q):
     result = ((q.high - q.low) / (p.high - p.low)).log()
     result[(q.low > p.low) | (q.high < p.high)] = float('inf')
     return result
 
 # Different distributions
+@register_kl(Bernoulli, Poisson)
+def _kl_bernoulli_poisson(p, q):
+    return -p.entropy() - (p.probs * q.rate.log() - q.rate)
```

Comment:
This uses 3 logarithms, and I think without using .entropy, we can make this 2 logarithms. What do you think?

Reply:
Same comment; either way is fine with me.